pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+
+<img className="w-1/3" src="https://media.giphy.com/media/5QMPpCzH6yxEqhev5A/giphy.gif" />
+
+In the end, we just wanted to make sure the initiative is community-driven, leaves no man behind. We tried to figure out a governance model that:
+- is as democratic as possible
+- supports the asynchronous decision-making process
+- gives power to people that "work", not companies that "pay". In other words, it gives equal power to both individual and corporate contributors.
+
+### Vocabulary
+
+In the governance model, you can find two essential roles, contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
+
+Committer rules in the repository, but for topics that go beyond, there is a Technical Steering Committee (TSC). 
+
+TSC consists of all committers. 
+
+Yes, not a dedicated group of people for now. Are you a committer? If yes, then it means you are a voting member of TSC either (unless you do not want to). But more on that further in the blog post.
+
+### Single TSC for spec and tools
+
+Spec and tools will have different licenses, but there will not be two TSCs though. One TSC for now to handle both specification and tools.
+
+Why?
+
+- start small, they say.
+- don't design processes and then work accordingly but better build a process that facilitates the way you work already.
+- we are a large community, but not large enough to handle two different charters, not now.
+
+The other good reason for not splitting is that we believe the initiative should work on spec and tools to provide essential open-source tools that consistently support the latest version of the specification. In other words, when we release the newest version of the specification, you can already use it with official tools and do not have to wait for other tooling providers to catch up. We know big companies do not care much about basic tooling as they most probably write their own as it is much easier for them to work without unnecessary noise. Nevertheless, we need to think about the small ones too. Two TSCs sound like having two silos that work separately, and it is not good for the start. Nobody says, though, that TSC can't organically grow into two separate groups in the future.
+
+
+### Value work more than money
+
+We want to have a TSC consisting of all the active committers in the project, not people who are there because their company is a sponsor of the project. You are TSC member because you work on the project, you code, write docs, maintain CI. You work, and therefore, you are a part of the initiative.
```

Comment:
```suggestion
We want to have a TSC consisting of all the active committers in the project, not people who are there because their company is a sponsor of the project. You are a TSC member because you work on the project, you code, write docs, maintain CI. You work, and therefore, you are a part of the initiative.
```